uwh-refbox/src/style.css

```diff
@@ -57,9 +59,9 @@ button:hover {
     -gtk-icon-shadow: none;
     -gtk-icon-effect: none;
 }
-button:disabled {
-    background-color: @grayed_out_gray;
-    color: @dark_gray;
+button*:disabled {
+    background-color: @disabled_gray;
+    color: @text_disabled_gray;
```

Comment:
This sets all descendants of the button to also have this style (due to the missing `.`). In this case that makes the labels in the +60, etc. buttons also have a background color, which makes it look as if the buttons no longer have rounded corners. This fixes:
```button:disabled,
button.little-blue:disabled {
    background-color: @disabled_gray;
    color: @text_disabled_gray;
}
button:disabled label {
    color: @text_disabled_gray;
}
```

Reply:
ok

Reply:
Your fix still doesn't solve the problem. Please implement above

Reply:
I did what you typed above, didn't fix the corner issue

Reply:
played around a little bit and got it working